### ISSUE TITLE
Feature Request: Trailers

### ISSUE THREAD

**Issue by jakob42** [dkanada/jellyfin-plugin-intros#15]
Hi,

I fear this might be out of scope for this plugin, but I would love to be able to view one or two movie trailers before starting the movie.

Thanks for the nice plugin!

**Comment by dkanada** [comment]
This plugin is only focused on adding prerolls, so it would indeed be out of scope. The feature you're looking for actually has an open pull request on the web client [here](https://github.com/jellyfin/jellyfin-web/pull/1178) if you'd like to track it.

**Comment by jakob42** [comment]
Wil do, thanks!
